wandb/sdk/launch/sweeps/launch_sweep.py

```diff
@@ -0,0 +1,294 @@
+import json
+from typing import Any, Dict, List, Optional, Union
+
+import click
+import yaml
+
+import wandb
+from wandb.apis import InternalApi, PublicApi
+from wandb.sdk.launch import utils as launch_utils
+from wandb.sdk.launch.sweeps.scheduler import Scheduler
+from wandb.sdk.launch.sweeps.utils import (
+    check_job_exists,
+    construct_scheduler_args,
+    get_previous_args,
+    handle_sweep_config_violations,
+)
+
+
+def launch_sweep(
+    entity: str,
+    project: str,
+    queue: str,
+    parsed_user_config: Optional[Dict[str, Any]] = None,
```

Comment:
seems unnecessarily verbose. this is public api, do we ever expect the user to have their config in serialized form?  IMO just call this `config` or `config_overrides`?